gazelle/python/testdata/binary_without_entrypoint_per_file_generation_partial_update/test.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 The Bazel Authors. All rights reserved.
```

Comment:
Please update copyright year to 2025.

Reply:
Done. FYI this was copied from another test, so I assume a lot of them are out of date.